ops/terraform/services/server/deploy.groovy

```diff
@@ -91,9 +91,9 @@ def runServerRegression(Map args = [:]) {
         'suite_version': 'v2',
         'spawn_rate': 10,
         'users': 10,
-        'spawned_runtime': '30s',
-        'compare_tag': "build${lastSuccessfulBuildID}__${sanitizedBranchName}",
-        'store_tag': "build${currentBuildId}__${sanitizedBranchName}"
+        'spawned_runtime': '1m',
```

Comment:
Curious if you could comment on the change to double the time. Was 30s insufficient? Obviously, if it's necessary, let's continue, otherwise I'm just concerned about adding another 30s to each environment's deployment (or 180s over our standard, monolithic, three-environment deployment).

Reply:
I recall seeing some inconsistent results with 30 seconds even after switching to the consistent dataset. I'll do some more runs against test with 30 seconds and see if I'm off on that.

Reply:
I've done a few 30 second runs, and I feel like we should be OK if we stick with `30s`. One of the runs _did_ hit warning thresholds for some of the more _flexible_ statistics (i.e. `0.99` response time percentiles), but has since been just fine and been fairly consistent.

I'll change it back to `30s` and if, in the future, we run into issues with that we can always increase it.

P.S. Maybe we should think about getting these values into SSM so we don't need to open a PR to adjust them in the future?